### PR TITLE
Update CHANGELOG.md, bump ssh-slaves plugin and update README.

### PR DESCRIPTION
**Pull request by ssk2**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changes
 
+## v3.0.2-2.32.3 (2017-03-27)
+  * Upgrades Jenkins to 2.32.3 LTS
+  * Include entire LD_LIBRARY_PATH in Docker CMD
+  * Updated default label to use 0.5.0-alpine of jenkins-dind
+  * Security improvement to remove service accounts credentials from environment
+  * Plugin upgrades to address [2017-03-20 security advisory](https://groups.google.com/forum/#!topic/jenkinsci-advisories/sHa2_nmmU0A).
```
Comment (colin-msphere) [inline CHANGELOG.md]: This is not in yet; need to update a few more.

Comment (ssk2) [inline CHANGELOG.md]: Done!

File: Dockerfile [hunk 2]
```diff
@@ -131,7 +131,7 @@ RUN /usr/local/bin/install-plugins.sh       \
   saml:0.12                      \
   scm-api:2.0.2                  \
   ssh-agent:1.13                 \
-  ssh-slaves:1.14                \
+  ssh-slaves:1.16                \
```
Comment (colin-msphere) [inline Dockerfile]: Needs update to `README`.

Comment (ssk2) [inline Dockerfile]: Is done.

**Pull request merged**